### PR TITLE
Press Enter in Password Field to Join Game

### PR DESCRIPTION
**Pull request by usairman82**
Hopefully this change will enable pressing enter from the password box while joining a protected game.

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
Hey hey! Welcome to the club!

So to get a dev environment set up, follow the steps in the Getting Started link in the Readme. The readme has the bare bones instructions but the Getting Started page is much more detailed. 

Next, when opening PRs, it’s best to title them appropriately. This one should be called “Press Enter in password field to join game” or something.

**Comment by usairman82** [comment]
@NoahTheDuke Sounds Great, I'll do that.

File: src/cljs/nr/gamelobby.cljs [hunk 1]
```diff
@@ -352,7 +352,11 @@
         [:p
          [:input.game-title {:on-change #(swap! s assoc :password (.. % -target -value))
                              :type "password"
-                             :value (:password @s) :placeholder "Password" :maxLength "30"}]]
+                             :value (:password @s) :placeholder "Password" :maxLength "30"
+                             :on-key-press (fn [e]
+                                            (when (= 13 (.charCode e))
+                                              ((join prompt))))}]]
```
Comment (NoahTheDuke) [inline src/cljs/nr/gamelobby.cljs]:
```suggestion
                                              (join prompt)))}]]
```
My apologies, I was unclear in my last suggestions.

Comment (usairman82) [reply src/cljs/nr/gamelobby.cljs]: No, I should have guessed that.  Thank you.

**Pull request merged**